start_proteus.py

```diff
@@ -462,8 +462,8 @@ def main():
                 solvevol_target[e] = esc_m
 
                 # print info to user
-                log.debug("    escape %s: m=%.2e kg,  dm=%.2e (%.3f%%)"%
-                                    (e, esc_m, esc_dm, 100*esc_dm/esc_m))
+                #log.debug("    escape %s: m=%.2e kg,  dm=%.2e (%.3f%%)"%
```

Comment:
These two lines could remain uncommented.

Reply:
The issue here was that these lines lead to a runtime error when remaining in. Can you confirm this @lsoucasse?